### DISCUSSION TITLE
Which mesh formats are supported?

### DISCUSSION THREAD

**Discussion by Suven-p** [google/brax#283] [Q&A] [answered]
Since, STL doesn't have colors, is it possible to add colors to a mesh? What other mesh formats does brax support?

**Comment by btaba** [comment, marked as answer]
Hi @Suven-p ! To add a color to a mesh, you can use the config here https://github.com/google/brax/blob/main/brax/physics/config.proto#L103-L104. Hope that helps!

**Reply by btaba** [reply]
As far as other mesh formats, obj works, anything that can be loaded by trimesh [here](https://github.com/google/brax/blob/main/brax/io/mesh.py#L50). We just don't load the textures currently
